inst/rmd/h/default.html

```diff
@@ -501,16 +439,24 @@
 $if(toc_float)$
 
 <!-- setup 3col/9col grid for toc_float and main content  -->
-<div class="row-fluid">
+<div class="row">
```

Comment:
@cpsievert was this change necessary for **bslib** and bootstrap 4 supports ? 

Asking because it changes some layout for some custom format based on `html_document()`. 
For example: https://github.com/Bioconductor/BiocStyle/issues/86 (see both doc examples paragraph width)

it is a small difference but I am wondering if this could impact other custom formats. So asking for your knowledge on this `row` vs `row-fluid`. 

Thanks.

Reply:
`.row-fluid` is a Bootstrap 2 class that changed to `.row` in Bootstrap 3 https://getbootstrap.com/docs/3.4/migration/

Reply:
Thanks. I understand now. 
This means that about the paragraph width change, this is because `html_document` was using an old class and no bootstrap CSS was used. Switching to BS3 `row-fluid` class makes this CSS rule to apply
```css
.row {
    margin-right: -15px;
    margin-left: -15px;
}
```
this lead to a change in style for all `html_document` output. 

A side effect of this PR. Glad I know about that now. I may document that in NEWS maybe.